src/nvim/ops.c

```diff
@@ -915,24 +915,44 @@ int do_record(int c)
       apply_autocmds(EVENT_RECORDINGENTER, NULL, NULL, false, curbuf);
     }
   } else {  // stop recording
+
```

Comment:
clint doesn't like this blank line
```suggestion
```

Reply:
fixed in f4720df882a3efd3abf58d1015120c0ac2538d76